modules/getting-started/pages/glossary.adoc

```diff
@@ -0,0 +1,203 @@
+= Anypoint Platform Glossary
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Authorized short versions of some product names are presented in parentheses, for example: Anypoint Design Center (Design Center).
+
+= A
```

Comment:
Would it make sense to set these as second level headers?
`== A` 
instead of
`= A`

Do you think it would improve navigation or term discoverability to have the letters showing up in the right navigation bar?

Reply:
Oh yes! Thanks for noticing the = headings aren't in right nav. Will fix!

Reply:
Also they should be == anyway! doh!